setup.py

```diff
@@ -363,13 +367,18 @@ def init_settings_from_config(self):
         )]
         if sys.platform.startswith('win') and sys.version_info < (3, 3):
             settings['include_dirs'].insert(0, pjoin('buildutils', 'include_win32'))
-        
+
+        settings.setdefault('libraries', [])
+        # Explicitly link dependencies, not necessary if zmq is dynamic
+        if sys.platform.startswith('win'):
+            settings['libraries'].extend(('ws2_32', 'iphlpapi', 'advapi32'))
```

Comment:
If this is only needed when libzmq is static, is it intentional that the linking unconditional?

Reply:
I just prefer it to be explicit. Easy to change, though.

Reply:
ok, thanks